pkgs/applications/networking/irc/senpai/default.nix

```diff
@@ -2,12 +2,12 @@
 
 buildGoModule rec {
   pname = "senpai";
-  version = "unstable-2023-02-13";
+  version = "0.2.0";
 
   src = fetchFromSourcehut {
     owner = "~taiite";
     repo = "senpai";
-    rev = "1318e784bd2bba3765aee97811a3f0053d3a6723";
+    rev = "v${version}";
     sha256 = "sha256-q167og8S8YbLcREZ7DVbJhjMzx4iO0WgIFkOV2IpieM=";
   };
 
```

Comment:
Please update meta.homepage to https://sr.ht/~taiite/senpai/, because the one provided below does not work.